test/fixtures/output/flink/shell/shell.golden

```diff
@@ -0,0 +1,232 @@
+]2;Statement successfully submitted.
+Statement phase is COMPLETED.
+configuration updated successfully.
++-----------------+---------------+
+|       Key       |     Value     |
++-----------------+---------------+
+| cli.another-key | another value |
++-----------------+---------------+
+ > set;> set;> set;
+]2;Statement successfully submitted.
+Statement phase is COMPLETED.
++------------------------+---------------------+
+|          Key           |        Value        |
++------------------------+---------------------+
+| cli.a-key              | a value             |
+| cli.another-key        | another value       |
+| client.service-account | <unset> (default)   |
+| sql.current-catalog    | default             |
+| sql.current-database   | db1                 |
+| sql.local-time-zone    | GMT+00:00 (default) |
++------------------------+---------------------+
+ > reset 'cli.another-key';> reset 'cli.another-key';> reset 'cli.another-key';
+]2;Statement successfully submitted.
+Statement phase is COMPLETED.
+configuration key "cli.another-key" has been reset successfully.
++------------------------+---------------------+
+|          Key           |        Value        |
++------------------------+---------------------+
+| cli.a-key              | a value             |
+| client.service-account | <unset> (default)   |
+| sql.current-catalog    | default             |
+| sql.current-database   | db1                 |
+| sql.local-time-zone    | GMT+00:00 (default) |
++------------------------+---------------------+
+ > set;> set;> set;
+]2;Statement successfully submitted.
+Statement phase is COMPLETED.
++------------------------+---------------------+
+|          Key           |        Value        |
++------------------------+---------------------+
+| cli.a-key              | a value             |
+| client.service-account | <unset> (default)   |
+| sql.current-catalog    | default             |
+| sql.current-database   | db1                 |
+| sql.local-time-zone    | GMT+00:00 (default) |
++------------------------+---------------------+
+ > reset;> reset;> reset;
+]2;Statement successfully submitted.
+Statement phase is COMPLETED.
+configuration has been reset successfully.
++------------------------+---------------------+
+|          Key           |        Value        |
++------------------------+---------------------+
+| client.service-account | <unset> (default)   |
+| sql.local-time-zone    | GMT+00:00 (default) |
++------------------------+---------------------+
+ > set;> set;> set;
+]2;Statement successfully submitted.
+Statement phase is COMPLETED.
++------------------------+---------------------+
+|          Key           |        Value        |
++------------------------+---------------------+
+| client.service-account | <unset> (default)   |
+| sql.local-time-zone    | GMT+00:00 (default) |
++------------------------+---------------------+
+ > exit> exit> exit
+]2;
+
+TEXT + ANSI ESCAPE SEQUENCES OUTPUT
```

Comment:
Is this really that useful? Personally, I just find it really confusing. If we make a PR that changes the output colors, wouldn't it be easier to manually verify the output?

Reply:
hmm yeah I kinda agree, I was also considering disabling colors since it will make the golden file easier to read